demo/ssn-input.html

```diff
@@ -20,6 +20,10 @@
         display: inline-block;
       }
 
+      :host([hidden]) {
+        display: none!important;
```

Comment:
No problem, I understand the value of uniform code style.  Updated commit running through CI now.